### PR TITLE
[libcorrect] Add a new port

### PR DESCRIPTION
**Pull request by netheril96**
**Describe the pull request**

- #### What does your PR fix?
Add library libcorrect

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
  all 
  no

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
Yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

**Comment by netheril96** [comment]
@microsoft-github-policy-service agree [company="Google"]

**Comment by netheril96** [comment]
@microsoft-github-policy-service agree company="Google"

**Review by MonicaLiu0311 — CHANGES_REQUESTED** [review]

File: ports/libcorrect/vcpkg.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "libcorrect",
+  "version-date": "2018-10-11",
+  "description": "libcorrect is a library for Forward Error Correction",
+  "homepage": "https://github.com/quiet/libcorrect",
```
[the remark this answers is not in the document]
Comment (netheril96) [reply ports/libcorrect/vcpkg.json:5]: Already done.

**Review by MonicaLiu0311 — APPROVED** [review]

**Comment by JavierMatosD** [comment]
Thank you!

**Pull request merged**